lang/factory.go

```diff
@@ -9,13 +9,16 @@ import (
 	"github.com/safedep/dry/log"
 )
 
-var languages = map[string]func() (core.Language, error){
-	"python": func() (core.Language, error) {
+var languages = map[core.LanguageCode]func() (core.Language, error){
+	core.LanguageCodePython: func() (core.Language, error) {
 		return NewPythonLanguage()
 	},
-	"javascript": func() (core.Language, error) {
+	core.LanguageCodeJavascript: func() (core.Language, error) {
```

Comment:
Changing this, so we have a single source of truth - language code in `core/language.go`